src/main.cpp

```diff
@@ -158,6 +158,14 @@ void copyFont(const QString& externalPathName, const QString& resourcePathName,
 
 int main(int argc, char* argv[])
 {
+    // print stdout to console if Mudlet is started in a console in Windows
```

Comment:
Isn't this a duplicate of #1998 ...?

Reply:
I've moved this functionality out into that PR.